packages/utils/src/__tests__/object.test.js

```diff
@@ -18,49 +18,42 @@ describe('object', () => {
     const testData = [
       [
         'should sort the keys of an object containing string values',
-        { fourth: 'd', third: 'c', second: 'b' },
-        { asc: true },
+        [{ fourth: 'd', third: 'c', second: 'b' }, { asc: true }],
```

Comment:
This is an example of grouping all input params together. In other words:
```js
[[input, options], expected]
```

instead of 
```js
[input, options, expected]
```

Both seem perfectly valid to me! But I slightly prefer the first because the separation between input and output is more obvious

Reply:
Yeah the first one is easier to read. I'll stick to the first format in the future conversions.